includes/admin/class-wc-payments-admin.php

```diff
@@ -1159,9 +1165,13 @@ private function get_uncaptured_transactions_count() {
 		$test_mode = WC_Payments::mode()->is_test();
 		$cache_key = $test_mode ? DATABASE_CACHE::AUTHORIZATION_SUMMARY_KEY_TEST_MODE : DATABASE_CACHE::AUTHORIZATION_SUMMARY_KEY;
 
+		$send_callback         = function() {
+			$request = Request::get( WC_Payments_API_Client::AUTHORIZATIONS_API . '/summary' );
+			return $request->send( 'wc_pay_get_authorizations_summary' );
```

Comment:
Would be like this too? 

```suggestion
			return $request->send( 'wc_pay_get_authorizations_summary' )->to_array();
```